.gitignore

```diff
@@ -43,3 +43,4 @@ coverage
 
 # cypress temp files
 cypress/videos
```

Comment:
can you add a newline to the end of the file? it's a tiny formatting thing but I hate the red circle that Git has 😆